op-devstack/dsl/proofs/super_cannon_helper.go

```diff
@@ -0,0 +1,19 @@
+package proofs
+
+import (
+	"github.com/stretchr/testify/require"
+
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-service/txintent/bindings"
+)
+
+type SuperCannonGameHelper struct {
+	*FaultDisputeGameHelper
+}
+
+func NewSuperCannonGameHelper(t devtest.T, require *require.Assertions, game *bindings.FaultDisputeGame) *SuperCannonGameHelper {
```

Comment:
This helper doesn't currently have any custom logic that differs from the `FaultDisputeGameHelper` utility that it wraps, but trying to roughly follow a similar model to `e2eutils`, and we can expect to add functionality here in the future.

Reply:
I hope we'll be able to hide the logic in `e2eutils` behind an interface. Then swap implementations once this helper is complete.

Reply:
Yeah, that makes sense.  This might be a little tricky, but I can look at trying to introduce some interfaces as a follow-up.  Easier to try to do this as we go, if possible.

Reply:
Add a follow-up ticket [here](https://github.com/ethereum-optimism/optimism/issues/16478)